tutorials/fmc/fmc.c

```diff
@@ -0,0 +1,64 @@
+#include "fmc.h"
+#include "libs/adc/api.h"
+#include "libs/timer/api.h"
+#include "tutorials/fmc/can_api.h"
+
+#include <avr/interrupt.h>
+#include <avr/io.h>
+#include <stdio.h>
```

Comment:
Is this import needed? You might need `<stdbool.h>` for `bool`...

Reply:
stdio would be for the int16_t right